ocp_utilities/infra.py

```diff
@@ -49,6 +53,30 @@ def assert_nodes_schedulable(nodes):
         )
 
 
+def assert_pods_failed_or_pending(pods: list) -> None:
```

Comment:
please get a ns as arg as well

Reply:
assert_pods_failed_or_pending(pods: list, namespace=None)
The original purpose is to validate all the cluster pods. So I will include the namespace as an optional arg.

Reply:
Namespace no longer: Reverted to the just the  `pods` as argument.

Reply:
`pods: list` -> `pods:list[Pod]`